recipes/leg2cheb/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "leg2cheb" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/Legendre-to-Chebyshev/archive/{{ version }}.tar.gz
+  sha256: 3ff70cecdb98cb851dc8b04709c616cdd6b3b681dee83c0c8b022a6422718189
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: 
+    - cd src
+    - meson setup build ${MESON_ARGS}
+    - meson compile -C build
+    - meson install -C build
+
+requirements:
+  build:
+    - numpy >=2.0
+    - meson                               
+    - ninja                               
+    - pkg-config         
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]            
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }} 
+    - {{ stdlib('c') }}
+
+  host:
+    - python
+    - cython
+    - fftw
+    - openblas
+    - libopenblas =*=*openmp*
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  run:
+    - numpy >=2.0 
```

Comment:
Numpy has a run_exports and will add the correct run_dependency if it is in host
```suggestion
```